tests/nightly/jenkins-nightly-run.sh

```diff
@@ -89,11 +89,15 @@ esac
 
 LOG_UPLOAD_DEST="${LOG_UPLOAD_DEST:-${DEFAULT_LOG_UPLOAD_DEST}}"
 
+DEFAULT_ARTIFACT_URL="https://storage.googleapis.com/vic-engine-builds/${input}"
```

Comment:
I like being able to completely override the artifact URL, but I would also parameterize the artifact bucket so that you don't always have to provide the full url including build numbers:
```bash
DEFAULT_ARTIFACT_BUCKET="vic-engine-builds"
ARTIFACT_BUCKET=${ARTIFACT_BUCKET:-${DEFAULT_ARTIFACT_BUCKET}}
...

# also need to update `gsutil ls` used for `input` with the ARTIFACT_BUCKET variable
constructed_url="https://storage.googleapis.com/${ARTIFACT_BUCKET}/${input}"
url="${ARTIFACT_URL:-${constructed_url}}"
...

wget -nv "${url}"
```

This would allow you to very easily pick up the latest build from a given bucket while still allowing the total override that you have in this PR already.